html/index.php

```diff
@@ -2,9 +2,38 @@
 
 require_once __DIR__.'/../vendor/autoload.php';
 
+
+$rows = [];
+$headers = [];
+$index = 0;
+
 $resultsStorage = new \ScraperBot\Storage\SqlLite3Storage('../railerdb.sqlite3');
 $crawls = $resultsStorage->getTimeStamps();
 
+$statusCodes = $resultsStorage->getStatusCodes();
+foreach ($statusCodes as $code) {
+    $headers[$index] = $code;
+    $statusResults = $resultsStorage->getStatsByStatus($code);
+    foreach ($statusResults as $timestamp => $statusResult) {
+        $index++;
+        $rows[$timestamp][$code] = $statusResult;
+    }
+
+}
+
+//print_r($rows);
+// Specify our Twig templates location
+$loader = new \Twig\Loader\FilesystemLoader(__DIR__.'/../src/templates');
+// Instantiate our Twig
+$twig = new \Twig\Environment($loader);
+$template = $twig->load('results_status.twig');
+echo $template->render(['headers' => $headers, 'rows' => $rows]);
```

Comment:
We should think about buffering output rather than using two separate echo statements.

Reply:
Won't we get the full html page output rendered twice in this case?